vim25/soap/client.go

```diff
@@ -43,48 +41,43 @@ type HasFault interface {
 }
 
 type RoundTripper interface {
-	RoundTrip(req, res HasFault) error
+	RoundTrip(ctx context.Context, req, res HasFault) error
 }
 
-var cn uint64 // Client counter
-
 type Client struct {
 	http.Client
 
-	u        url.URL
-	insecure bool
-	_        uint32 // for alignment: https://code.google.com/p/go/issues/detail?id=6404
-
-	cn  uint64         // Client counter
-	rn  uint64         // Request counter
-	log io.WriteCloser // Request log
+	u url.URL
+	k bool // Named after curl's -k flag
```

Comment:
Still need a 32-bit field next to the bool for alignment.  Or, I think you can move the bool to the end of the struct (after all the other 64-bit fields / aligned structs).